drush.make

```diff
@@ -257,13 +257,13 @@ libraries[civihr_tasks][overwrite] = TRUE
 libraries[org.civicrm.shoreditch][destination] = modules/civicrm/tools/extensions
 libraries[org.civicrm.shoreditch][download][type] = git
 libraries[org.civicrm.shoreditch][download][url] = https://github.com/civicrm/org.civicrm.shoreditch
-libraries[org.civicrm.shoreditch][download][tag] = v0.1-alpha26
+libraries[org.civicrm.shoreditch][download][tag] = v0.1-alpha28
 libraries[org.civicrm.shoreditch][overwrite] = TRUE
 
 libraries[org.civicrm.styleguide][destination] = modules/civicrm/tools/extensions
 libraries[org.civicrm.styleguide][download][type] = git
 libraries[org.civicrm.styleguide][download][url] = https://github.com/civicrm/org.civicrm.styleguide
-libraries[org.civicrm.atyleguide][download][tag] = v0.1-alpha6
+libraries[org.civicrm.atyleguide][download][tag] = v0.1-alpha7
```

Comment:
Seems there's an error in the spelling s/org.civicrm.atyleguide/org.civicrm.styleguide

Reply:
Nice catch! And it seems it has been like that for a while :(